src/test/java/com/nanalysis/sharepoint/ManualTests.java

```diff
@@ -72,6 +77,15 @@ public void uploadFile() throws Exception {
         System.out.println("File created");
     }
 
+    @Ignore("Manual test")
+    @Test
+    public void uploadBigFile() throws Exception {
+        File f = new File(BIG_FILE_LOCAL_PATH);
+        Consumer<Double> progress = percent -> System.out.printf("Uploading: %.2f%%%n", percent);
+        client.uploadBigFile(TEST_PATH, f.getName(), f.length(), new FileInputStream(f), progress);
+        System.out.println("File created");
+    }
```

Comment:
How to test that?

Reply:
Changing constants on top to target a site that you have access to (and not using OAUTH2 if using your personnal credentials) should be enough.